### ISSUE TITLE
Failed to activate the MavensMate-Atom package

### ISSUE THREAD

**Issue by scameirano** [atom/atom#6227]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.189.0
**System**: Mac OS X 10.9.5
**Thrown From**: Atom Core

### Stack Trace

Failed to activate the MavensMate-Atom package

```
At literal is not a function

[stdin]:4:13: error: literal is not a function
  'themes': [
            ^
```

### Commands

```
     -1:24.2 core:backspace (atom-text-editor.editor.mini)
     -1:22.1 core:confirm (atom-text-editor.editor.mini)
```

### Config

```json
{}
```

### Installed Packages

```coffee
# User
MavensMate-Atom, v0.0.20

# Dev
No dev packages
```

**Comment by lee-dohm** [comment]
Probably related to #6226

**Comment by mnquintana** [comment]
This was probably resolved along with #6226 in https://github.com/atom/atom/commit/06b8195fb8de1fe21755f1469004d11498280aec
